### ISSUE TITLE
Weitere Optimierung der Events-Abfrage 

### ISSUE THREAD

**Issue by ChrisMalewski** [52North/wv-eventing-rest-api#35]
Aktuelle Laufzeit ist ok.
Wie skaliert die Laufzeit?
Gemeinsame Diskussion über Optimierungsmöglichkeiten.

**Comment by ChrisMalewski** [comment]
WV liefert aktuellen Dump. 
52N analysiert Query Laufzeiten. 
Danach weitere Diskussion.

**Comment by matthesrieke** [comment]
Carsten hat die Abfrage soweit momentan möglich optimiert (u.a. durch Herausnahme der Subscription-Referenzen). In unserer Entwicklungs-Umgebung liegen wir im Schnitt bei ca 3 sec für die Abfrage der (ersten) 100 Events des Nutzers.

@ChrisMalewski : Aus unserer Sicht haben wir keine weiteren Hebel, ist das soweit in Ordnung?

**Comment by ChrisMalewski** [comment]
@matthesrieke 
Ja, das ist völlig in Ordnung. Wir müssen das ganze im Prod. System testen und ggf. auch noch die DB neu konfigurieren -> siehe Performanze der Sensor Web Observation API!
